src/main/java/TaskTypeEnum.java

```diff
@@ -0,0 +1,63 @@
+import java.util.HashMap;
+import java.util.Map;
+
+public enum TaskTypeEnum {
+
+    TODO(0, "T"),
+    DEADLINE(1, "D"),
+    EVENT(2, "E");
+
+    private Integer typeId;
+
+    private String typeShortForm;
+
+    TaskTypeEnum(final Integer typeId, final String typeShortForm) {
```

Comment:
Should the variable name be in SCREAMING_SNAKE case, because it's a constant?

Reply:
Yes, for enum class this will be the constant value, so it will be in caps.